pilot/pkg/networking/core/configgen.go

```diff
@@ -44,6 +44,6 @@ type ConfigGenerator interface {
 }
 
 // NewConfigGenerator creates a new instance of the dataplane configuration generator
-func NewConfigGenerator(plugins []string) ConfigGenerator {
-	return v1alpha3.NewConfigGenerator(registry.NewPlugins(plugins))
+func NewConfigGenerator(plugins []string, proxyProtocol bool) ConfigGenerator {
```

Comment:
That seems a bit extreme.. What is so special about proxyProtocol - that no other option in istio has - to have it as  a param to this method ?

Reply:
@costinm agree, I'll switch to a less intrusive technique.  Thanks for the feedback, that's really why I opened the PR, so I could get some pointers on a better way to push down the configuration.

Reply:
updated, or rather, reverted.